File: tags/REL-1.0/src/Servidor/Transaccional/AccesoDatos/Dao/NHibernate/DaoDetallePagoNHibernate.cs
using NLog;
using Trascend.Bolet.ObjetosComunes.Entidades;
using Trascend.Bolet.AccesoDatos.Contrato;

namespace Trascend.Bolet.AccesoDatos.Dao.NHibernate
{
    class DaoDetallePagoNHibernate : DaoBaseNHibernate<DetallePago, string>, IDaoDetallePago
    {
        private static Logger logger = LogManager.GetCurrentClassLogger();
    }
}

File: src/Servidor/Transaccional/AccesoDatos_Fac/Dao/NHibernate/DaoDetallePagoNHibernate.vb
Imports NLog
Imports Diginsoft.Bolet.AccesoDatos.Contrato
Imports Trascend.Bolet.ObjetosComunes.Entidades
Imports Trascend.Bolet.AccesoDatos.Dao.NHibernate
Namespace Dao.NHibernate
    Public Class DaoDetallePagoNHibernate
        Inherits DaoBaseNHibernate(Of DetallePago, String)
        Implements IDaoDetallePago
        Private Shared logger As Logger = LogManager.GetCurrentClassLogger()
    End Class
End Namespace